site/data/notifications.yml

```diff
@@ -3,7 +3,7 @@ notifications:
     message: >-
       Register to join us online for our open development planning sessions
       bi-weekly, starting Wednesday, July 19th, 8am-9am PT!
-    published: true
+    published: false
```

Comment:
I'd like to keep this published for now while we're kickstarting this bi-weekly meeting.

Reply:
does this need still persist now @erquhart?

Reply:
I would think so, since it alerts people from the home page, rather than requiring people to go specifically to the Community page for more info. We're starting to get more participants, but it's still pretty small.